libtraceevent/parse-filter.c

```diff
@@ -1624,9 +1624,11 @@ int pevent_filter_clear_trivial(struct event_filter *filter,
 		case FILTER_TRIVIAL_FALSE:
 			if (filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
 		case FILTER_TRIVIAL_TRUE:
 			if (!filter_type->filter->boolean.value)
 				continue;
+			/* fall through */
```

Comment:
Right. I think so, too.. Use `break` instead of two the fall-through comment ?
```
diff --git a/libtraceevent/parse-filter.c b/libtraceevent/parse-filter.c
index 185774a..feb68e1 100644
--- a/libtraceevent/parse-filter.c
+++ b/libtraceevent/parse-filter.c
@@ -1624,11 +1624,11 @@ int pevent_filter_clear_trivial(struct event_filter *filter,
                 case FILTER_TRIVIAL_FALSE:
                         if (filter_type->filter->boolean.value)
                                 continue;
-                        /* fall through */
+                        break;
                 case FILTER_TRIVIAL_TRUE:
                         if (!filter_type->filter->boolean.value)
                                 continue;
-                        /* fall through */
+                        break;
                 default:
                         break;
                 }
```